src/rows.jl

```diff
@@ -227,6 +230,28 @@ function Base.show(io::IO, row::Row)
     return nothing
 end
 
+"""
+    schema_field_names(::Type{<:Legolas.Schema})
+
+Get a tuple with the names of the fields of this `Legolas.Schema`, including names that
+have been inherited from this `Legolas.Schema`'s parent schema.
+"""
+schema_field_names(::Type{S}) where {S<:Legolas.Schema} = throw(UnknownSchemaError(S()))
+schema_field_names(s::Legolas.Schema) = schema_field_names(typeof(s))
+schema_field_names(::Legolas.Row{S}) where {S} = schema_field_names(S)
+schema_field_names(::Type{<:Legolas.Row{S}}) where {S} = schema_field_names(S)
```

Comment:
```suggestion
schema_field_names(::Type{S}) where {S<:Schema} = throw(UnknownSchemaError(S()))
schema_field_names(s::Schema) = schema_field_names(typeof(s))
schema_field_names(::Row{S}) where {S} = schema_field_names(S)
schema_field_names(::Type{<:Row{S}}) where {S} = schema_field_names(S)
```
There's no need to qualify things with the name of the module you're currently in (except in very rare circumstances).